test.sh

```diff
@@ -4,11 +4,11 @@ set -e
 # a helper script to run tests
 
 if ! which nsqd >/dev/null; then
-    echo "missing nsqd binary" && exit 1
+  echo "missing nsqd binary" && exit 1
 fi
 
 if ! which nsqlookupd >/dev/null; then
-    echo "missing nsqlookupd binary" && exit 1
+  echo "missing nsqlookupd binary" && exit 1
```

Comment:
please do not change the 4-space indent to 2-space indent in the shells scripts

(4 space indent is the style we use for shell, and also it unnecessarily increases the diff to review)